README.md

```diff
@@ -53,6 +53,7 @@ circuitpython-build-tools package.
 ```shell
 python3 -m venv .env
 source .env/bin/activate
+pip install wheel
```

Comment:
Would it be easier/better to add `wheel` to requirements.txt?

Reply:
Good idea. Will do that.